impersonation/README.md

```
@@ -0,0 +1,92 @@
+
+  - rds-session: `<JWT>`
+
+- **Request Body:**
+
+- Body Parameters:
+
+  - `impersonatedUserId`: **Required.** String. The userId of the user to be impersonated.
+  - `reason`: **Required.** String. The reason for requesting impersonation.
+
+  - Example Request Body:
+    ```json
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)